apps/core/src/components/Validator.tsx

```diff
@@ -38,10 +44,14 @@ export function Validator({
         isApyApproxZero,
         validatorSummary,
         system,
+        isPendingValidators,
     } = useValidatorInfo({
         validatorAddress: address,
     });
 
+    if (isPendingValidators) {
+        return <div className="flex items-center justify-center">...</div>;
```

Comment:
We can use Skeleton here, wdyt?
```suggestion
        return <Card>
                <CardImage shape={ImageShape.SquareRounded}>
                    <div className="h-10 w-10 animate-pulse bg-neutral-90 dark:bg-neutral-12" />
                    <Skeleton widthClass="w-10" heightClass="h-10" isRounded={false} />
                </CardImage>
                <div className="flex flex-col gap-y-xs">
                    <Skeleton widthClass="w-40" heightClass="h-3.5" />
                    <Skeleton widthClass="w-32" heightClass="h-3" hasSecondaryColors />
                </div>
                <div className="ml-auto flex flex-col gap-y-xs">
                    <Skeleton widthClass="w-20" heightClass="h-3.5" />
                </div>
            </Card>;
```

Reply:
sounds good